arcade/tilemap/tilemap.py

```diff
@@ -102,6 +102,16 @@ def _get_image_source(
     return None
 
 
+def _may_be_flip(tile, texture):
```

Comment:
This should have type definitions, like:

```py
def _may_be_flip(tile: pytiled_parser.Tile, texture: arcade.Texture) -> arcade.Texture:
    ...
```

Reply:
Ok, I've added them.

Thanks for the review.